explorer/next-sitemap.config.mjs

```diff
@@ -0,0 +1,11 @@
+/** @type {import('next-sitemap').IConfig} */
+const siteMapConfig = {
+  changefreq: "monthly",
+  generateIndexSitemap: false,
+  outDir: "./out/explore",
```

Comment:
@BruceRodrigues the outDir depends on the basePath as well should be process.env.NEXT_PUBLIC_BASE_PATH || "/explore"

Reply:
oh right! Sorry about that